responses/12_phase-3-config.md

```diff
@@ -0,0 +1,27 @@
+# Config logic
+
+To put this response in the config, it will be very similar to phase 1. But, _where_ the action goes is different. We need to be careful of the gate here. If the gate fails, we can have special logic for the "unhappy path" response. The "happy path" response will be a regular response triggered if the gate is successful, like:
+
+```
+- type: gate
```

Comment:
I definitely agree these should be the same files 👍 

I'm not sure about having it be the same left/right parameters as the previous example. While it would keep the thread more consistent in terms of the example, we aren't specifying that the user should instruct their learners to title the pull request a certain thing. I'd rather keep the flexibility in what we allow the authors to ask their learners to do, and this logic will probably fit more use cases.

I also thought about changing the previous example to match this one for consistency, but I like the idea of introducing another example of a gate here. I think exposing the authors to different possibilities is a feature here, not a bug. What do you think?

I addressed the file naming in 59c95c5.